solc/main.cpp

```diff
@@ -50,8 +50,16 @@ static void setDefaultOrCLocale()
 #endif
 }
 
+struct T {
+	~T() {
+		cout << "Hit <ENTER> to BREXIT\n";
```

Comment:
I guess this is not supposed to be here ;)

Reply:
irks, I'll squash it away upon next push.